streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -816,6 +817,106 @@ public void shouldCleanAndReviveCorruptedStandbyTasksBeforeCommittingNonCorrupte
         verify(consumer);
     }
 
+    @Test
+    public void shouldCloseAndReviveUncorruptedTasksWhenTimeoutExceptionThrownFromCommit() {
```

Comment:
The test says: should _close and revive_

How do we exactly verify this? Maybe we do, but it's not clear to me from the code. Can you elaborate?

Reply:
We verify the revive by asserting that it went from `RUNNING` back to `CREATED`